### PR TITLE
🐛 Fix application host switch

### PR DESCRIPTION
**Pull request by dominikkaminski**
**Changes:**

1. Add host_array variable
2. Replace host with submitted value in error message

## How can others test the changes?

```shell
npm start -- --host=0.0.0.0
```

## PR-Checklist

Please check the boxes in this list after submitting your PR:

- [x] You can merge this PR **right now** (if not, please prefix the title with "WIP: ")
- [x] I've tested **all** changes included in this PR.
- [x] I've also reviewed this PR myself before submitting (e.g. for scrambled letters, typos, etc.).
- [x] I've rebased the `develop` branch with my branch before finishing this PR.
- [x] I've **summarized all changes** in a list above.
- [x] I've mentioned all **PRs, which relate to this one**.
- [x] I've prefixed my Pull Request title is according to [gitmoji guide](https://gitmoji.carloscuesta.me/).

## Example

<details>
<summary>
:warning: Before merging please click here to expand and follow the guide.
</summary>
<br>

Please use `:twisted_rightwards_arrows:` at the beginning of your merge commit title.

Example 1:

<pre>
<code>
:twisted_rightwards_arrows: :bug: Fix Wrong Text Decoration at ...
</code>
</pre>

To get your commit message, just copy the first part of this pull request.

Example 2:
<pre>
<code>
**Changes:**

- Fixes Wrong Text Decoration at ...
- Fixes some typos
- ...

**Issues:**

Closes #NumberOfFixedIssue

PR: #NumberOfThisPR
</code>
</pre>
</details>

## Emojis

<details>
<summary>
Expand for a list of most used Emojis.
</summary>
<br>

Please prefix your commit messages with an Emoji.

Ref: https://gitmoji.carloscuesta.me/

| Description              | Glyphe               | Emoji  |
|--------------------------|----------------------|--------|
| Bugfix                   | `:bug:`              | 🐛     |
| Configuration releated   | `:wrench:`           | 🔧     |
| Cosmetic                 | `:lipstick:`         | 💄     |
| Dependencies Downgrade   | `:arrow_down:`       | ⬇️     |
| Dependencies Upgrade     | `:arrow_up:`         | ⬆️     |
| Formatting               | `:art:`              | 🎨     |
| Improving Performance    | `:zap:`              | ⚡️     |
| Initial commit           | `:tada:`             | 🎉     |
| Linter                   | `:rotating_light:`   | 🚨     |
| Miscellaneous            | `:package:`          | 📦     |
| New Feature              | `:sparkles:`         | ✨     |
| Refactoring Code         | `:recycle:`          | ♻️     |
| Releasing / Version tags | `:bookmark:`         | 🔖     |
| Removing Stuff           | `:fire:`             | 🔥     |
| Tests                    | `:white_check_mark:` | ✅     |
| Work In Progress (WIP)   | `:construction:`     | 🚧     |

</details>

### CODE REVIEW

File: bin/bpmn-studio.js [hunk 1]
```diff
@@ -56,10 +56,11 @@ function _applicationHostIsValid(host) {
...
+  const host_array = host.split('.');
   const addressHasNotFourOctetts = host_array.length !== 4;
   if (addressHasNotFourOctetts) {
-    console.log("Host is not a valid ip address [0.0.0.0].\n");
+    console.log(host + " is not a valid ip address [0.0.0.0].\n");
     return false;
   }
   return true;
-}
+}
```
Comment (Vyperus) [inline bin/bpmn-studio.js]: Missing new line.

**Comment by dominikkaminski** [comment]
@Vyperus New line added.

I've added `"files.insertFinalNewline": true` Visual Studio Code to prevent this from happening again.

**Comment by dominikkaminski** [comment]
@cmg-dev I've updated error message according to your suggestion.

File: bin/bpmn-studio.js [hunk 2]
```diff
@@ -56,9 +56,10 @@ function _applicationHostIsValid(host) {
   if (host === null || host === undefined) {
     return false;
   }
+  const host_array = host.split('.');
   const addressHasNotFourOctetts = host_array.length !== 4;
   if (addressHasNotFourOctetts) {
-    console.log("Host is not a valid ip address [0.0.0.0].\n");
+    console.log("The specified host: '"+ host +"' is not a valid IP address, like: '0.0.0.0'.\n");
```
Comment (LeoTT) [inline bin/bpmn-studio.js]: Would use template string.
```ts
`The specified host: '${host}' is not a valid.......`
```

File: bin/bpmn-studio.js [hunk 3]
```diff
@@ -56,9 +56,10 @@ function _applicationHostIsValid(host) {
   if (host === null || host === undefined) {
     return false;
   }
+  const host_array = host.split('.');
   const addressHasNotFourOctetts = host_array.length !== 4;
```
Comment (LeoTT) [inline bin/bpmn-studio.js]: How about using a regex instead?
`/^\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}$/` should do the trick.

**Pull request merged**